### PR TITLE
Fix GoogleStorageDAO NPE

### PR DESCRIPTION
**Pull request by rtitle**
I encountered this in Leo tests.

**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

After merging, _even if you haven't bumped the version_: 
- [ ] Update `README.md` and the `CHANGELOG.md` for any libs you changed with the new dependency string. The git hash is the same short version you see in GitHub (i.e. seven characters). You can commit these changes straight to develop.

### CODE REVIEW

**Review by helgridly — APPROVED** [review]
how sneaky

File: google/src/main/scala/org/broadinstitute/dsde/workbench/google/HttpGoogleStorageDAO.scala [hunk 1]
```diff
@@ -187,9 +187,9 @@ class HttpGoogleStorageDAO(appName: String,
     val objects = listObjectsRecursive(getter) map { pagesOption =>
       pagesOption.map { pages =>
         pages.flatMap { page =>
-          Option(page.getItems.asScala) match {
+          Option(page.getItems) match {
             case None => List.empty
-            case Some(objects) => objects.toList
+            case Some(objects) => objects.asScala.toList
```
Comment (jmthibault79) [inline google/src/main/scala/org/broadinstitute/dsde/workbench/google/HttpGoogleStorageDAO.scala:198]: Good catch!  But let's rename this or the other `objects` in line 187

Comment (rtitle) [reply google/src/main/scala/org/broadinstitute/dsde/workbench/google/HttpGoogleStorageDAO.scala:198]: Ah good call, yeah I don't like it when variables shadow other variables

**Comment by helgridly** [comment]
@rtitle what's the status on this?

**Comment by rtitle** [comment]
Oh I forgot about this. It fixes a NPE if you call `listObjectsWithPrefix` on an empty bucket. Leo tests call this method on the staging bucket, which is never empty, so it works. I'd still like to merge this however, in case anyone else runs into it.

**Pull request merged**